app/code/community/RicardoMartins/PagSeguro/Block/Form/Cc.php

```diff
@@ -17,7 +17,15 @@ class RicardoMartins_PagSeguro_Block_Form_Cc extends Mage_Payment_Block_Form_Cc
     protected function _construct()
     {
         parent::_construct();
-        $this->setTemplate('ricardomartins_pagseguro/form/cc.phtml');
+
+        if($this->helper("ricardomartins_pagseguro")->isMultiCcEnabled())
+        {
+            $this->setTemplate('ricardomartins_pagseguro/form/multi-cc.phtml');
+        }
+        else
```

Comment:
else não é necessário ao manter $this->setTemplate('ricardomartins_pagseguro/form/cc.phtml').

Reply:
De acordo. Acabou que esta lógica foi movida pra outra área - removida desta função e, por isso, não encontrará a correção desta questão neste ponto do código.